### ISSUE TITLE
Add MailPoet integration where recipients can reply to comment

### ISSUE THREAD

**Issue by tevyaw** [websharks/comment-mail#218]
I think you could compete with Postmatic really well if people could reply to MailPoet messages (that send the post to their inbox) and it would add those as comments on the post, like Postmatic does, but even better since you can control what your posts look like, though MailPoet.

**Comment by jaswsinc** [comment]
@tevyaw Thanks for the feedback! :-) It sounds like a cool idea to me also.

Just to make sure I have it right...

What you're suggesting is some sort of integration that would allow for replies to an email which contains a Post Notification, to be sent back to the site as a comment on that Post. Correct?

Hmm. It is beginning to look like Comment Mail's RVE handler might be more useful overall than we first thought it would be—once it can support some additional twists like this one.

**Comment by tevyaw** [comment]
Yes. Postmatic sends out your posts to subscribers, and if they reply to the email, it adds their reply as a comment on that post. However, it takes away the control and statistics, and such that you get via MailChimp or MailPoet. So I'm suggesting that if people use MailPoet and Comment Mail together, you provide the same functionality, but with more ability to customize the automatic newsletter via MailPoet, but with the same result: people can reply to the newsletter email and have that posted as a comment. I'm not totally sure it really realistic, in a situation when you don't have control over the email being sent like Postmatic does. But I thought I'd suggest it anyway, and maybe your brilliant minds would be able to come up with a great solution.
